src/store.rs

```diff
@@ -281,4 +287,42 @@ mod tests {
 
         assert_eq!(updated_ticket.status, Status::Done)
     }
+
+    #[test]
+    fn add_comment_to_ticket() {
+        //arrange
+        let mut ticket_store = TicketStore::new();
+        let ticket = generate_and_persist_ticket(&mut ticket_store);
+        //act
+        let result = ticket_store.add_comment_to_ticket(ticket.id, "Test Comment".to_string());
+        //assert
+        assert!(result.is_some());
+        let ticket = ticket_store.get(ticket.id).unwrap();
+        assert_eq!(ticket.comments.len(), 1);
+    }
+
+    #[test]
+    fn add_comment_to_invalid_ticket_id_returns_none() {
+        let faker = fake::Faker;
+
+        //arrange
+        let mut ticket_store = TicketStore::new();
+
+        //act
+        let result = ticket_store.add_comment_to_ticket(faker.fake(), faker.fake());
+
+        //assert
+        assert!(result.is_none());
+    }
+
+    #[test]
+    #[should_panic(expected = "Comment cannot be empty")]
```

Comment:
I didn't know you could assert on the panic message, neat 💯

Reply:
👍 This test has been removed now as it no longer needs to test that the  add_comment_to_ticket function is panicking as that now takes a Comment structure.